### ISSUE TITLE
composite flowchart does not render properly

### ISSUE THREAD

**Issue by k-byun** [benweet/stackedit#1697]
Just made up the term "composite flowchart" but the mermaid code below:

```
graph LR
A-->B-->C
```

does not get drawn properly.  Tried this out on Atom with success.

**Comment by b4r** [comment]
Which version of mermaid are you running?
From https://github.com/mermaid-js/mermaid/issues/916 and https://github.com/mermaid-js/mermaid/releases/tag/8.2.6 the feature was requested and added.
If you run `npm update` from the stackedit root does the update help fix?

**Comment by k-byun** [comment]
Version 2.0.0
[cid:image001.png@01D6D83C.E0AE8760]

**Comment by b4r** [comment]
Oh - I'm sorry - so it's the Chrome Extensions. Is the StackEdit version you're using also the Chrome Extension?

**Comment by k-byun** [comment]
Yes, I'm using the Chrome Extension. 
I didn't know if there was another way to install it - I will explore this side this afternoon.  Looked like there is another way to install by using "git clone ..."

**Comment by b4r** [comment]
It's no problem - just leaving note here that I'll try and research the extensions (I broke my existing install of the nodejs bundle/build) but if anyone else has insight on this it would of course be appreciated.

**Comment by b4r** [comment]
I think I got it updated finally.  Steps to reproduce updating to the latest `mermaid@8.8.4` as of today in order to use the chaining graph feature.

1. `git clone https://github.com/benweet/stackedit.git`
   * This makes our new repo - I would advise cloning to a new directory for testing / confirming this "fix" ...
2. `cp updated-stackedit__package.json stackedit/package.json`
3. `cp updated-stackedit__.babelrc stackedit/.babelrc`
4. `cd stackedit`
   * We should now have the `stackedit` repo primed for updates.
5. Run `npm install` (similar as original `README.md` instructed by the repo).
   * This fails for me regarding `gulp build-prism` so if it fails for you run `npm install gulp`.
   * Now run `npm install` again to complete.
6. ** One last prep for `mermaid@8.8.4` extracted from the tarball (see the updated `package.json` you'll see we're using the github tarball).
   * For this step - if you're still inside the `stackedit` root directory - `cd node_modules/mermaid`.
   * Install `jison` in some way.
   * Here's the trick, compile the `.jison` files to `.js` with the `jison` tool.
      * Here's a little help to scan through the current `node_modules/mermaid` directory we just `cd`'d to  :  
         ````bash
         #!/bin/bash
         here=`pwd`;
         find . -iname '*.jison' | while read line; do
             echo ${line%/*};
             echo $line;
             cd ${line%/*};
             #jison ${line##*/} ;
             cd $here;
         done
         ````
   * Uncomment the `jison ${line##*/};` line when ready.
   * `cd ../../` back to the `stackedit` root directory.
7. Now finally `npm start`.

So far this process works for me.

Working on the chrome extension part next.

**Comment by k-byun** [comment]
Looking forward for the chrome extension.  Thanks!

**Comment by b4r** [comment]
Repo for the chrome extension https://github.com/b4r/stackedit-chrome-extension-project

Please let me know if I missed something.

**Comment by dayfuaim** [comment]
I confirm it doesn't working properly:
![Снимок экрана от 2021-03-02 17-32-30](https://user-images.githubusercontent.com/1274699/109664236-fcf56080-7b7d-11eb-8f7d-095509a785c4.png)

I use web version https://app.stackedit.io. No additional extensions.

**Comment by b4r** [comment]
Correct.  The web version from app.stackedit.io does **not** seem to currently use the updated [mermaid v8.2.6](https://github.com/mermaid-js/mermaid/releases/tag/8.2.6) version which allows chaining to work properly.

**Comment by benweet** [comment]
Fixed with v5.14.10.
